prql-compiler/prqlc/Cargo.toml

```diff
@@ -8,10 +8,6 @@ repository.workspace = true
 rust-version.workspace = true
 version.workspace = true
 
-[lib]
-doctest = false
```

Comment:
The reason I added these around is that the creating the binaries & linking them takes the majority of the time on incremental builds — in particular when running tests when watching the path. And so following https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html advice, it makes sense to disable these if we're not using them